src/SimpleVideoCutter/ChooseOutputDirectoryDialog.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.IO;
+using System.Windows.Forms;
+using System.Linq;
+
+namespace SimpleVideoCutter
+{
+    public partial class ChooseOutputDirectory : Form
```

Comment:
(I report it here, but this comment applies also to other places) 
I have recently cleaned up development branch from any compiler warnings. In your branch I see 133 warnings. For example here: 

```
Severity	Code	Description	Project	File	Line	Suppression State
Warning (active)	CS8618	Non-nullable field 'originalOutputDir' must contain a non-null value when exiting constructor. Consider declaring the field as nullable.	SimpleVideoCutter	ChooseOutputDirectoryDialog.cs	24	

```

Reply:
I cleaned them up in my fork. Though I did not count them, I made much less fixes than 133 (probably between 10 and 20).

The only messages I see now are related to duplicate names in `FormSettings.resx`, and there are 118 of them. Could they have been introduced by my IDE by accident? I did not consciously edit that file.

Excerpt:
```
1>FormSettings.resx : warning MSB3568: Duplicate resource name ">>buttonCancel.Name" is not allowed, ignored.
1>FormSettings.resx : warning MSB3568: Duplicate resource name ">>buttonCancel.Type" is not allowed, ignored.
1>FormSettings.resx : warning MSB3568: Duplicate resource name ">>buttonCancel.Parent" is not allowed, ignored.
1>FormSettings.resx : warning MSB3568: Duplicate resource name ">>buttonCancel.ZOrder" is not allowed, ignored.
...
```

Reply:
It turned out that those resource names were there before my changes. Can I safely delete them?